src/RHOCore.js

```diff
@@ -85,7 +85,7 @@ function toJSData(par /*: IPar */) /*: Json */{
         return ex.g_bool;
       }
       if (typeof ex.g_int !== 'undefined') {
-        return ex.g_int;
+        return parseInt(ex.g_int, 10); // ISSUE: overflow
```

Comment:
What's the overflow issue? Integer overflow is documented as well as any other feature of rholang. I don't think JS should try to change that behavior.

Reply:
Protobuf returns int64 as string, presumably because js numbers can
only handle 53 bits. For example, 2^60. It fits fine in a protobuf int64 but in a javascript number, it won't be exact.

IOU a better comment. _I explained it in the commit message of 7d9e32f, but that's not where it belongs._